### ISSUE TITLE
Symbols `__atomic_XXX` not found in i686-linux-android target

### ISSUE THREAD

**Issue by dangeross** [breez/breez-sdk-greenlight#1141]
Symbols relating to atomics are not being found for the `i686-linux-android` target:

```
Caused by: java.lang.UnsatisfiedLinkError: Unable to load library 'breez_sdk_bindings':
    dlopen failed: cannot locate symbol "__atomic_fetch_add_8" referenced by "/data/app/com.cBreez.client.debug-1/base.apk!/lib/x86/libbreez_sdk_bindings.so"...
    dlopen failed: cannot locate symbol "__atomic_fetch_add_8" referenced by "/data/app/com.cBreez.client.debug-1/base.apk!/lib/x86/libbreez_sdk_bindings.so"...
    dlopen failed: cannot locate symbol "__atomic_fetch_add_8" referenced by "/data/app/com.cBreez.client.debug-1/base.apk!/lib/x86/libbreez_sdk_bindings.so"...
    Native library (android-x86/libbreez_sdk_bindings.so) not found in resource path (.)
```

This seems to originate from the rust-openssl crate where versions > 0.10.57 (openssl-sys > 0.9.92) where there is a mishandling of atomics for i686, and pinning the version to the following works:
```
openssl = { version = "=0.10.57", features = ["vendored"] }
openssl-sys = "=0.9.92"
```

https://github.com/sfackler/rust-openssl/issues/2163
https://github.com/sfackler/rust-openssl/issues/2043

**Comment by JssDWt** [comment]
Given the sdk has a 'soft' dependency to 0.10, clients have the option to downgrade openssl themselves. Perhaps we can document this issue for 32 bit devices and recommend developers to downgrade openssl themselves?

**Comment by dangeross** [comment]
> Given the sdk has a 'soft' dependency to 0.10, clients have the option to downgrade openssl themselves. Perhaps we can document this issue for 32 bit devices and recommend developers to downgrade openssl themselves?

@JssDWt This would only work for Rust clients right?, not the bindings

**Comment by JssDWt** [comment]
> @JssDWt This would only work for Rust clients right?, not the bindings

True, they would have to either build the bindings themselves, or replace the underlying sdk binaries.
